src/modules/packagechooser/PackageChooserViewStep.h

```diff
@@ -67,6 +68,8 @@ class PLUGINDLLEXPORT PackageChooserViewStep : public Calamares::ViewStep
     PackageChooserMode m_mode;
     QString m_id;
     CalamaresUtils::Locale::TranslatedString* m_stepName;  // As it appears in the sidebar
+    QString m_description;
+    int m_defaultIdx;
```

Comment:
Perhaps this one should be a string, rather than an int, so it can be matched to the *id* of one of the items rather than their presentation order?